proc/proc.go

```diff
@@ -28,7 +28,7 @@ func GetKallsyms(kallsymsPath string) (*libpf.SymbolMap, error) {
 	var address uint64
 	var symbol string
 
-	symmap := libpf.SymbolMap{}
+	symmap := libpf.NewSymbolMap(200 * 1024)
```

Comment:
Should this be commented that kernel easily exposes 160k+ text symbols (that's what I have on my desktop).